tmt/steps/provision/podman.py

```diff
@@ -151,6 +153,24 @@ def start(self):
             ['--name', self.container, '-v', f'{workdir}:{workdir}:Z',
              '-itd', self.image])[0].strip()
 
+    def reboot(self, hard=False, container_name=None):
+        if not self.podman:
+            self.debug('container', 'No podman instance provided.', 'red')
+            return
+
+        self.info('container', 'Rebooting via podman', 'green')
+        container_name_win = container_name
+        if self.container:
+            container_name_win = self.container
+
+        if hard:
+            self.podman(['container', 'stop', container_name_win])
+            self.podman(['container', 'start', container_name_win])
+        else:
+            self.podman(['container', 'restart', container_name_win])
```

Comment:
Why not calling `self.stop()` and `self.start()` methods here?

Reply:
Ok. Need to take a look at podman again.
For me: stop -> shut down container
start -> start container 
restart -> doing by not stopping container and just somehow reboot.

There is slightly difference in semantic for me (start/stop manipulates with container state but reboot just doing something to let machine reboot (like send ACPI request for reboot). But I need to take a look at this again.

Reply:
Ok. According docs, it just doesn't matter:
The restart command allows containers to be restarted using their ID or name. Containers will be stopped if they are running and then restarted. Stopped containers will not be stopped and will only be started.

Which leads me to think about: how to implement hard reboot but I think this will be ok for now.

Reply:
Should be solved.